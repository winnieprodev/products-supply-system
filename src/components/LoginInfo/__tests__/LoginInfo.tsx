import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";

import LoginBox from "../LoginInfo";

describe("LoginBox", () => {
  describe("Snapshot tests", () => {
    test("it matches the snapshot", () => {
      const { container } = render(
        <BrowserRouter>
          <LoginBox />
        </BrowserRouter>,
      );
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
