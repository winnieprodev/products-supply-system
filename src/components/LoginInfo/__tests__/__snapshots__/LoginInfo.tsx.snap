// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginBox Snapshot tests it matches the snapshot 1`] = `
.c0 {
  position: relative;
  display: inline-block;
  z-index: +1;
}

.c1 {
  cursor: pointer;
}

.c2 {
  width: 24px;
  height: 24px;
  border-radius: 50%;
  margin-left: 24px;
  text-align: center;
  line-height: 24px;
  background: center no-repeat url(test.jpg);
  background-size: contain;
  border: 1px solid white;
  color: #FFFFFF;
}

<div
  class="c0"
>
  <div
    class="c1"
  >
    <div
      class="c2"
    />
  </div>
</div>
`;
